sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/encoder/jsonencoder/_utils.py

```diff
@@ -21,7 +21,7 @@
 try:
     import jsonschema
 except ImportError:
-    jsonschema = None
+    raise ImportError("install jsonschema to use this feature") from None
```

Comment:
Assuming this is a test change to trigger APIView step for schemaregistry.

Reply:
Yes, just a temporary change.

Reply:
Everything looks good, so reverting the temporary changes.